doc/screen-recording.md

```diff
@@ -74,16 +90,19 @@ which means you will need to combine the streams. Here's an example of that.
 
 ``` ts
 // set up needed events
-sdk.on('sessionStart', async (session) => {
+sdk.on('sessionStarted', async (session) => {
```

Comment:
There are still two instances of `'sessionStart'` above in screen-recording.md. (lines 27 and 58).